src/Utilities.js

```diff
@@ -56,6 +56,24 @@ export function isConnected(node) {
   return !!(current && (current.__CE_isImportDocument || current instanceof Document));
 }
 
+/**
+ * @param {!DocumentFragment} fragment
+ * @return {!Array<!Element>}
+ */
+export function childrenFromFragment(fragment) {
+  // Note, IE doesn't have `children` on document fragments.
+  if (fragment.children) {
+    return Array.prototype.slice.call(fragment.children);
```

Comment:
Would we get anything from reading `fragment.children` once? IDK, not really important.

Reply:
Sure.